bears/general/FilenameBear.py

```diff
@@ -71,6 +77,18 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if not filename_without_extension.startswith(prefix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} prefix.'.format(prefix),
+                file=filename)
+
+        if not filename_without_extension.endswith(suffix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} suffix.'.format(suffix),
+                file=filename)
```

Comment:
If you suggest patches, we maybe need to take care if a file has not both: correct prefix and suffix. Then we would get a merge conflict for the patches^^ We could merge the results into a single result and merge the messages together (depending what combination we get).